Makefile

```diff
@@ -142,15 +142,15 @@ wasm-sdk-e2e-test: generate
 .PHONY: check
 check:
 ifeq ($(DOCKER_RUNNING), 1)
-	docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:${GOLANGCI_LINT_VERSION} golangci-lint run -v
+	docker run --rm -v $(shell pwd):/app:Z -w /app golangci/golangci-lint:${GOLANGCI_LINT_VERSION} golangci-lint run -v
```

Comment:
Really? I wouldn't expect the linter to write files

Reply:
The linter wants to `stat` the working directory and fails:

```
$ make check
docker run --rm -v /home/zregvart/tmp/opa:/app -w /app golangci/golangci-lint:v1.51.0 golangci-lint run -v
level=warning msg="[config_reader] Can't make abs path for \"./...\": stat .: permission denied"
level=info msg="[config_reader] Config search paths: [./ . /root]"
ERROR 2023/06/29 11:35:31 could not discover absolute path: stat .: permission denied
ERROR 2023/06/29 11:35:31 could not discover absolute path: stat .: permission denied
level=warning msg="Failed to discover go env: failed to run 'go env': exit status 1"
level=info msg="[lintersdb] Active 7 linters: [errcheck gosimple govet ineffassign staticcheck typecheck unused]"
level=info msg="[loader] Go packages loading at mode 575 (deps|exports_file|name|types_sizes|compiled_files|files|imports) took 2.118453ms"
level=error msg="Running error: context loading failed: failed to load packages: failed to load with go/packages: err: exit status 1: stderr: cannot determine current directory: stat .: permission denied\n"
level=info msg="Memory: 2 samples, avg is 31.4MB, max is 31.4MB"
level=info msg="Execution took 5.143799ms"
make: *** [Makefile:145: check] Error 3
```

Reply:
Added the read-only (`ro`) flag to the linter invocation...